src/main/java/frc/robot/Subsystem/Shooter/Shooter.java

```diff
@@ -0,0 +1,141 @@
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
+  private void upConfigs(){
+    var talonFXConfigs = new TalonFXConfiguration();
+    // set slot 0 gains
+    var slot0Configs = talonFXConfigs.Slot0;
+    slot0Configs.kS = UP_KS; // Add 0.25 V output to overcome static friction
+    slot0Configs.kA = UP_KA; // An acceleration of 1 rps/s requires 0.01 V output
+    slot0Configs.kP = UP_KP; // An error of 1 rps results in 0.11 V output
+    slot0Configs.kI = UP_KI; // no output for integrated error
+    slot0Configs.kD = UP_KD; // no output for error derivative
+
+    // set Motion Magic Velocity settings
+    var motionMagicConfigs = talonFXConfigs.MotionMagic;
+    motionMagicConfigs.MotionMagicAcceleration = ACCELERATION; // Target acceleration of 400 rps/s (0.25 seconds to max)
+    motionMagicConfigs.MotionMagicJerk = JERK; // Target jerk of 4000 rps/s/s (0.1 seconds)
+
+    //Limits
+    var limitConfigs = new CurrentLimitsConfigs();
+    // enable stator current limit
+    limitConfigs.StatorCurrentLimit = CURRENT_LIMIT;
+    limitConfigs.StatorCurrentLimitEnable = true;
+
+    //upload configs to motor
+    StatusCode statusCode = StatusCode.StatusCodeNotInitialized;
+    for (int i = 0; i < 5; i++){
+      statusCode = m_upMotor.getConfigurator().apply(talonFXConfigs);
+      if (statusCode.isOK())
+        break;
+    }
+    if (!statusCode.isOK())
+      System.out.println("Shooter Arm could not apply config, error code:" + statusCode.toString());
+  }
+  private void downConfigs(){
+    var talonFXConfigs = new TalonFXConfiguration();
+    // set slot 0 gains
+    var slot0Configs = talonFXConfigs.Slot0;
+    slot0Configs.kS = DOWN_KS; // Add 0.25 V output to overcome static friction
+    slot0Configs.kA = DOWN_KA; // An acceleration of 1 rps/s requires 0.01 V output
+    slot0Configs.kP = DOWN_KP; // An error of 1 rps results in 0.11 V output
+    slot0Configs.kI = DOWN_KI; // no output for integrated error
+    slot0Configs.kD = DOWN_KD; // no output for error derivative
+
+    // set Motion Magic Velocity settings
+    var motionMagicConfigs = talonFXConfigs.MotionMagic;
+    motionMagicConfigs.MotionMagicAcceleration = ACCELERATION; // Target acceleration of 400 rps/s (0.25 seconds to max)
+    motionMagicConfigs.MotionMagicJerk = JERK; // Target jerk of 4000 rps/s/s (0.1 seconds)
+
+    //Limits
+    var limitConfigs = new CurrentLimitsConfigs();
+    // enable stator current limit
+    limitConfigs.StatorCurrentLimit = CURRENT_LIMIT;
+    limitConfigs.StatorCurrentLimitEnable = true;
+
+    //upload configs to motor
+    StatusCode statusCode = StatusCode.StatusCodeNotInitialized;
+    for (int i = 0; i < 5; i++){
+      statusCode = m_downMotor.getConfigurator().apply(talonFXConfigs);
+      if (statusCode.isOK())
+        break;
+    }
+    if (!statusCode.isOK())
+      System.out.println("Shooter Arm could not apply config, error code:" + statusCode.toString());
+ }
```

Comment:
why is the configs not at the bottom?

also, if you copy-paste can you at least delete the unrelated comments